Update 103-data-transformation.md (#5855)
kamranahmedse/developer-roadmap: src/data/roadmaps/data-analyst/content/107-data-cleaning/103-data-transformation.md

# Data Transformation 

> Data Transformation, also known as Data Wrangling, is an essential part of a Data Analyst's role. This process involves the conversion of data from a raw format into another format to make it more appropriate and valuable for a variety of downstream purposes such as analytics. Data Analysts transform data to make the data more suitable for analysis, ensure accuracy, and to improve data quality. The right transformation techniques can give the data a structure, multiply its value, and enhance the accuracy of the analytics performed by serving meaningful results.
> 
> - [@article@What is Data Transformation - Geeks for Geeks](https://www.geeksforgeeks.org/what-is-data-transformation/)

Reviewer: ```suggestion
Data Transformation, also known as Data Wrangling, is an essential part of a Data Analyst's role. This process involves the conversion of data from a raw format into another format to make it more appropriate and valuable for a variety of downstream purposes such as analytics. Data Analysts transform data to make the data more suitable for analysis, ensure accuracy, and to improve data quality. The right transformation techniques can give the data a structure, multiply its value, and enhance the accuracy of the analytics performed by serving meaningful results.

Learn more from the following resources:

- [@article@What is Data Transformation?](https://www.geeksforgeeks.org/what-is-data-transformation/)
```

Reviewer: You mention in your PR description that you've added links to Python and R examples but I don't see them?

# Data Transformation 

Data Transformation, also known as Data Wrangling, is an essential part of a Data Analyst's role. This process involves the conversion of data from a raw format into another format to make it more appropriate and valuable for a variety of downstream purposes such as analytics. Data Analysts transform data to make the data more suitable for analysis, ensure accuracy, and to improve data quality. The right transformation techniques can give the data a structure, multiply its value, and enhance the accuracy of the analytics performed by serving meaningful results.

> - [@article@What is Data Transformation - Geeks for Geeks](https://www.geeksforgeeks.org/what-is-data-transformation/)

Reviewer: ```suggestion
- [@article@What is Data Transformation](https://www.tibco.com/glossary/what-is-data-transformation)
```

> - [@article@Example with Python](https://www.geeksforgeeks.org/time-series-data-transformation-using-python/)

Reviewer: ```suggestion
- [@article@Transforming Data with Python](https://www.udacity.com/blog/2023/01/transforming-data-with-python.html)
```

> - [@article@Example with R](https://www.geeksforgeeks.org/data-cleaning-transformation-with-dplyr-in-r/)

Reviewer: ```suggestion
- [@article@Data Transformation with R](https://r4ds.had.co.nz/transform.html)
```
